Feature main

enaburekhan opened the pull request.

enaburekhan committed:
creation and styling the nav bar

enaburekhan committed:
added main structure and the styling

enaburekhan committed:
refactored the reference source font inside head

enaburekhan committed:
added the avatar icon

enaburekhan committed:
added images to the aside element

enaburekhan committed:
re-aligning the numbers of views section

enaburekhan committed:
updated the read me file

enaburekhan committed:
re-edited the read me file

enaburekhan committed:
refactored the CSS and readme file

ktyyctrk committed:
modified the subscribe button

ktyyctrk committed:
moved the style of body from  html to css

ktyyctrk committed:
added comment  on css

ktyyctrk committed:
Added comment on html file

ktyyctrk committed:
modified the readme file

ktyyctrk committed:
Changed the title

ktyyctrk committed:
modified the style of <ul< under main

enaburekhan committed:
added comments to html

enaburekhan committed:
added comments to CSS

ktyyctrk committed:
screenshot added

kitpao commented:
# 👋 Hello @enaburekhan and @ktyyctrk 

First of all, welcome to Microverse! 😄 I hope you enjoy your learning path here 👍 

:pencil: Friendly reminder: Remember it's important to have a professional repo so you can increase your job opportunities.
Please take a step back and read the lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
Sadly, for now, your review has been marked as **invalid** because of the following reason:
- [ ] You are missing a short summary of your changes in the pull request.

Note:

- [ ] For next time, remember to write your commits in the imperative tense, as stated in [the article pointed in pathwright](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)

Once you correct this, please submit a new code review and we will be glad to check it :)
### ⚠️ Status: Your project has been marked as invalid ⚠️ 
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

enaburekhan commented:
**Project 1: Embedding videos and images**

- Embedded a video player 
- Added the associated thumbnail images on the aside section
- Added the respective styling

MuriloRoque commented on style.css:
```diff
@@ -0,0 +1,167 @@
+/* removes text under-line */
+A {
```
- [ ] This should be lowercase, right? :sweat_smile:

MuriloRoque commented on style.css:
```diff
@@ -0,0 +1,167 @@
+
+
+.nav-bar {
+    display: flex;
```
You are repeating a lot of `display: flex` properties throughout your CSS. Please, make a `flex` class with this property and apply it in the 'flexed' elements you want, this will make your CSS a lot more readable. Keep this in mind for future projects :wink:

MuriloRoque commented on style.css:
```diff
@@ -0,0 +1,167 @@
+  
+}
+
+.video-img-1 {
+  display: flex;
+  align-items: center;
+}
+
+.video-img-2 {
+  display: flex;
+  align-items: center;
+}
+
+
+.video-img-3 {
+  display: flex;
+  align-items: center;
+}
+
+.video-img-4 {
+  display: flex;
+  align-items: center;
+}
+
+.video-img-5 {
+  display: flex;
+  align-items: center;
+}
+
+.video-img-6 {
+  display: flex;
+  align-items: center;
+}
+
+.video-img-7 {
+  display: flex;
+  align-items: center;
+}
+
```
All of these elements have the same properties. You can actually select them all and apply this rule only once!

MuriloRoque reviewed (changes requested):
# Status: Changes requested :red_circle: 

## Hello @enaburekhan and @ktyyctrk :wave: 

## Welcome to Microverse, guys, I am sure you will love it! You have done a very good job in this project, but we still need to make some changes, alright? Here are my suggestions:

# Page

## Navigation Bar

- [ ] Your Youtube logo should be an embedded image. Currently, it's an icon with a text beside it ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/navbar_logo_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

![Screenshot from 2020-08-24 14-06-32](https://user-images.githubusercontent.com/59691171/91074851-8813fd80-e613-11ea-9dfd-8ed4e76ceebe.png)

## Video Information Section

- [ ] Please, make an effort to make this section look more like the original one (like the video information text for example), check below:

### Yours:

![Screenshot from 2020-08-24 14-17-03](https://user-images.githubusercontent.com/59691171/91075703-db3a8000-e614-11ea-83f9-1a035de96515.png)

### Original:

![Screenshot from 2020-08-24 14-18-14](https://user-images.githubusercontent.com/59691171/91075744-e9889c00-e614-11ea-9bdc-e2d016afa7f4.png)

<br>

## Recommended Videos Section

- [ ] Please, make an effort to make this section look more like the original one (like the autoplay and the user who uploaded it for example), check below:

### Yours:

![Screenshot from 2020-08-24 14-17-06](https://user-images.githubusercontent.com/59691171/91075803-fc02d580-e614-11ea-8d07-c43ce23a9795.png)

### Original:

![Screenshot from 2020-08-24 14-18-17](https://user-images.githubusercontent.com/59691171/91075818-01f8b680-e615-11ea-823f-ad7452ef7c0f.png)

<br>

# General

- [ ] We have some issues in your HTML, especially when opening and closing < a > tags, check [this validator](https://validator.w3.org), and get rid of all errors and warnings, please.

- [ ] Please check my comments in your code below

- [ ] Please add your Live Link to the README file, you can use [Raw Githack](https://raw.githack.com/) for that.

- [ ] The last TSE mentioned you should give a brief description in your Pull Request, you can actually edit the description instead of making a comment, so please try doing that before the next review :smiley: 

# Summary

- Your page is very nicely implemented. Congratulations :clap: :clap: :clap: 
- You still need to change somethings in your page layout;
- You still need to fix some code, I wrote comments on it to help you!

## Kindly make the requested changes and submit for another review, please :pray: 

## I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[![Twitter Follow](https://img.shields.io/twitter/follow/MuriloRoquePai1?label=Follow%20Murilo%20on%20Twitter&style=social)](https://twitter.com/MuriloRoquePai1)

ktyyctrk committed:
Add Side Content İmage

ktyyctrk committed:
Fix Navbar and aside bar

enaburekhan committed:
fix video title

enaburekhan committed:
fix subscribers section

ktyyctrk committed:
Fix Autoplay and Background

enaburekhan committed:
refactor Html/Css

ktyyctrk committed:
Add screenshot

enaburekhan committed:
added live demo link

enaburekhan commented:
- Fix Youtube logo in navigation bar
- Replace the thumb images in the aside bar 
- Added the auto-play button
- Fix the subscribe button
- Fix the readme demo link
- Refactor html and Css codes

kubilaycaglayan reviewed (changes requested):
Hello Team :wave: 
Status: Changes Required :recycle: 

Welcome to the Microverse!  :star:  :star:  :star:  :star:  :star: 
We are so happy to see you at our school.
I wish you a lot of success throughout your experience...

I loved your project! :green_heart: 
It looks really great! :boom: 

Let's cover some details to make your project in mastery level, together :arrow_down: 

**Linters**

:bulb: Every one of our projects in Microverse will use linters. We are doing this check with GitHub Actions(GA). It checks your code against syntax errors. And when looking at your PR(Pull Request), I see your checks are failed. You can see the details by clicking on it. 

- [ ]  Please fix your linter errors.

**Video Information Section**

- [ ]  The requirements of the previous TSE still apply, I am sure you can do better to make it look like the original. :muscle: 

**Recommended Videos Section**

A nice improvement on this part :+1: 

- [ ]  Please, work on it one more time, you can practice the distance between elements and make the links clickable. :blush: 

**HTML Validation**

- [ ]  There are fewer errors but we don't want any of them. Please use the W3 HTML Validator to see at which parts of your code is giving errors and kindly fix them :pray:  

**CSS**

:bulb: As the previous TSE mentioned, you are repeating the **display: flex** property a lot of times. So, what to do?

I will try to show it :arrow_down: 
- If I want to use the same CSS rule on a lot of different elements, I can create a class selector:
```css
.make-this-element-flex {
  display: flex;
}
```

- And then you can give this class name to any element that you want to have **display: flex;** property :arrow_down: 

```html
<div class="make-this-element-flex"> ....... </div>
```

- And as you know you can use this class name with other class names :arrow_down: 

```css
.make-this-element-red {
  color: red;
}
```

```html
<div class="make-this-element-flex make-this-element-red"> ....... </div>
```

- Better naming :arrow_down: 

```css
.flex {
  display: flex;
}

.red {
  color: red;
}
```

```html
<div class="flex red"> ....... </div>
```

As a result, you have a **div** with red font color and have flex property. 

- [ ]  Now you can collect your d-flex properties in one selector.

That's all. I hope this helps you :pray: 

With all my respect...

enaburekhan committed:
refactor CSS display flex

enaburekhan committed:
fix aside images

ktyyctrk committed:
Fix Video information section

enaburekhan committed:
fix semantic errors

enaburekhan committed:
fix html

enaburekhan commented:
**Code review:**

- Fix the video section
- Fix the recommended video(images) section
- Fix the display flex property
- Fix html errors
- Linter is not part of the first project
![image](https://user-images.githubusercontent.com/51296741/91297673-a0605580-e796-11ea-8ceb-d4367dd00327.png)

kubilaycaglayan reviewed (changes requested):
Hello Team :wave: 
Status: Changes Required :recycle: 

Great job on implementing the suggested changes! :star: 

**Correction:** My last review on this repository has two misleading statements. Please consider taking my apologies. :pray: **The linters and the HTML validation** is not yet required for this first project. Surely, you don't have to do anything about it(linters and validation) for this project. This information has changed recently, so I have just learned it.

### Let's move on to the **Recommended Videos Section**
Here's a :camera_flash:  **Screenshot**
![rightpadding](https://user-images.githubusercontent.com/60448833/91331379-5ea1d000-e7d3-11ea-90b5-492b55a048d8.png)

#### Two things going on here :arrow_down: 
**First:**

- [ ]  Reduce the right-padding on the recommended videos section please

**Second:**

- [ ]  To align "Up next" text on the same line with the video visuals, you can apply some padding.

Other than that, the project looks so much better! 
### That's it. After doing these changes you can request for another code review!

You can reach out to me through Slack...
With all my respect... :pray:

enaburekhan committed:
fix styling errors

enaburekhan commented:
**Code review:**

- Fix right padding on recommended video section
- Align "Up next"

santiagorodriguezbermudez commented on index.html:
```diff
@@ -0,0 +1,152 @@
+                </figure> 
+                
+              </li>
+
```
- [ ] Add a hr tag or a border to mimic the original page. Take a look at the original page, it has a division between the first video and the following ones:
![Screen Shot 2020-08-27 at 6 37 23 AM](https://user-images.githubusercontent.com/5817056/91437705-ecaba400-e82f-11ea-80cd-eaf06345c830.png)

santiagorodriguezbermudez reviewed (changes requested):
## Status: Changes Required  :recycle: ##

Hey guys! Good work working on the mentioned changes by the last TSE. However, we must still work on the page to look better. 

## Improve the following:

- [ ] Your page is missing the footer. Kindly implement the footer according to the original page. 
![Screen Shot 2020-08-27 at 6 15 51 AM](https://user-images.githubusercontent.com/5817056/91436301-75751080-e82d-11ea-8e87-99634295c4be.png)

- [ ] Update the link of your DEMO so that always displays the latest changes. For this, use the 'development' link provided by githack instead of the 'production' link. 

- [ ] Add padding between the side videos and the main content as requested by the previous TSE. Right now, it looks a bit messy. 
<img width="1367" alt="Screen Shot 2020-08-27 at 6 21 58 AM" src="https://user-images.githubusercontent.com/5817056/91436402-a35a5500-e82d-11ea-8f31-5fa382143e01.png">

- [ ] Video stats are not the same in terms of layout and positioning. Kindly work on these elements to make it similar. For example, your 'subscribe' button is not in the same position as the one in the project. 
### Yours:
<img width="982" alt="Screen Shot 2020-08-27 at 6 27 52 AM" src="https://user-images.githubusercontent.com/5817056/91436806-72c6eb00-e82e-11ea-9af5-93aa5512fdca.png">

### Project:
![Screen Shot 2020-08-27 at 6 27 32 AM](https://user-images.githubusercontent.com/5817056/91436783-66db2900-e82e-11ea-9c51-ffde1fc1db66.png)

### Please implement the mentioned changes and submit a CR afterward :nerd_face:

If you have any questions, please reach out on slack (@santiagorodriguezb) :nerd_face:

ktyyctrk committed:
Fix Side Container

ktyyctrk committed:
Modify ReadMe

ktyyctrk committed:
Live Link Added

ktyyctrk committed:
Fix Live Demo Link

ktyyctrk committed:
add line after first video

ktyyctrk commented:
**Code review:**

- Fix right container padding and line
- Update Demo link
- Add the reference video page screensot in Readme file for comparison of video stats part.
- Footer is not within the requirements of the project

kubilaycaglayan reviewed (approved):
Hello Team :wave: 
## Status: Project Approved :green_circle: 

### Congratulations on finishing your first project. :clap:  :clap: 
### Welcome to the Microverse! A great journey is waiting for you.

With all my respect... :pray:

enaburekhan committed:
css style not needed

enaburekhan committed:
fix styling

enaburekhan committed:
fix styling issue
